### ISSUE TITLE
Allow user to be auto-assigned to own backport PRs

### ISSUE THREAD

**Issue by watson** [sqren/backport#188]
It would be nice if it was possible to be assigned to the backport PRs that you open.

I usually have a list open of PR's assigned to me, and sometimes I forget to merge the backport PRs I've opened because they are not assigned to me.

I'm not sure if everyone wants this or if this should be a personal setting though.

**Comment by sqren** [comment]
Makes sense. WDYT of the following:

As a configuration option (in `.backport/config.json`
```json
{
  "targetPRAssignees": ["watson"]
} 
```

As a cli argument. If nothing is given it'll default to the current user:
```
backport --assign
```

Or assign a specific person:
```
backport --assign sqren
```

**Comment by watson** [comment]
I like it. I'd personally vote for making the json option singular like the CLI option, e.g. `targetPRAssign`. It can still be an array of course.

**Comment by sqren** [comment]
Thanks for the feedback @watson ! The reason I went with assignees is because that's what Github is calling it:

![image](https://user-images.githubusercontent.com/209966/83250225-13622c00-a1a8-11ea-8745-6164fca94e51.png)

https://developer.github.com/v3/issues/assignees/#add-assignees-to-an-issue

I agree that `targetPRAssignees` doesn't read well. What I mean is: the assignees for the target PR. It could also simply be:
```json
{
  "assignees": []
}
```

Or

```json
{
  "assigneesForTargetPR": []
}
```

Or

```json
{
  "assigneesForTargetPullRequest": []
}
```

**Comment by sqren** [comment]
If we prefer `assign` as verb instead of `assignee` as a noun it could be:

```json
{
  "assignToTargetPullRequest": []
}
```

**Comment by watson** [comment]
Ah didn't think of what GitHub called it. I'll leave this up to you 😉

**Comment by sqren** [comment]
Released in [v5.4.3](https://www.npmjs.com/package/backport/v/5.4.3)
